cob_bringup/robots/cob3-2.xml

```diff
@@ -0,0 +1,177 @@
+			<arg name="robot" value="$(arg robot)" />
+		</include>
+		<include file="$(find cob_bringup)/controllers/base_controller.launch" >
+			<arg name="robot" value="$(arg robot)" />
+		</include>
+		
+		<include file="$(find cob_bringup)/tools/base_collision_observer.launch" >
+			<arg name="robot" value="$(arg robot)" />
+		</include>
+
+		<include file="$(find cob_bringup)/drivers/battery_voltage_filter.launch" >
+			<arg name="robot" value="$(arg robot)" />
+		</include>
+		<!-- upper robot -->
+		<include file="$(find cob_bringup)/drivers/schunk_powercube_chain_driver.launch" >
+			<arg name="component_name" value="torso" />
+			<arg name="robot" value="$(arg robot)" />
+		</include>
+		<include file="$(find cob_bringup)/controllers/cob_trajectory_controller.launch">
+			<arg name="robot" value="$(arg robot)"/>
+			<arg name="component_name" value="torso"/>
+		</include>
+		<include file="$(find cob_bringup)/drivers/schunk_powercube_chain_driver.launch" >
+			<arg name="component_name" value="tray" />
+			<arg name="robot" value="$(arg robot)" />
+		</include>
+		<include file="$(find cob_bringup)/controllers/cob_trajectory_controller.launch">
+			<arg name="robot" value="$(arg robot)"/>
+			<arg name="component_name" value="tray"/>
+		</include>
+		<include file="$(find cob_bringup)/drivers/schunk_powercube_chain_driver.launch" >
+			<arg name="component_name" value="arm" />
+			<arg name="robot" value="$(arg robot)" />
+		</include>
+		<include file="$(find cob_bringup)/controllers/cob_trajectory_controller.launch">
+			<arg name="robot" value="$(arg robot)"/>
+			<arg name="component_name" value="arm"/>
+		</include>
+		<include file="$(find cob_bringup)/drivers/schunk_sdh.launch" >
+			<arg name="robot" value="$(arg robot)" />
+		</include>
+
+		<include file="$(find cob_bringup)/drivers/head_axis.launch" >
+			<arg name="robot" value="$(arg robot)" />
+		</include>
+
+		<!-- start additional packages -->
+		<include file="$(find cob_bringup)/tools/diagnostics_aggregator.launch" >
+			<arg name="robot" value="$(arg robot)" />
+		</include>
+		<include file="$(find cob_bringup)/tools/teleop.launch" >
+			<arg name="robot" value="$(arg robot)" />
+			<arg name="version" value="v1" />
+		</include>
+		<include file="$(find cob_bringup)/drivers/openni2.launch" >
+			<arg name="robot" value="$(arg robot)" />
+			<arg name="name" value="cam3d"/>
+			<arg name="device_id" value="#1"/>
+		</include>
+		<!--include file="$(find cob_bringup)/drivers/image_flip_nodelet.launch" >
+			<arg name="robot" value="$(arg robot)" />
+			<arg name="nodelet_manager" value="cam3d_nodelet_manager"/>
+			<arg name="start_manager" value="false"/>
+		</include-->
+		<include file="$(find cob_cam3d_throttle)/launch/cam3d_throttle.launch" >
+			<arg name="namespace" value="/cam3d" />
+			<arg name="rate" value="1.0" />
+			<arg name="start_manager" value="false"/>
+			<arg name="nodelet_manager" value="cam3d_nodelet_manager"/>
+		</include>
```

Comment:
Pc1 is right , should be on pc2, but pc2 has only usb3.0 ports and this kinect version only works if it is attached to a usb2.0 ... I haven't got other posibility